### DISCUSSION TITLE
root directory of the app

### DISCUSSION THREAD

**Discussion by zzzgit** [coqui-ai/TTS#2491] [General Q&A]
<img width="225" alt="image" src="https://user-images.githubusercontent.com/1060733/230728134-7aa66e96-8a9f-4f91-8d68-78d1c5200bc2.png">

But where is the root directory?

**Comment by p0p4k** [comment]
What root?

**Reply by zzzgit** [reply]
> What root?

I failed to download the default model due to a network problem. It caused me couldn't use TTS anymore, so I need to delete the whole model(which failed to be completely downloaded), I think.
I don't know which folder should I delete.

**Reply by p0p4k** [reply]
It's root of your Linux, models download under cache I think.

**Comment by zzzgit** [comment]
**~/Library/Application Support/tts**

It's here.
